spyder/plugins/ipythonconsole/widgets/kernelconnect.py

```diff
@@ -140,15 +140,45 @@ def __init__(self, parent=None):
         self.accept_btns.accepted.connect(self.accept)
         self.accept_btns.rejected.connect(self.reject)
 
+        # save kernel checkbox
+        self.save_kernel_layout = QCheckBox(self)
```

Comment:
It would be nice if the save connection check box was inline with the cancel/ok buttons. 

I think you should be able to stick them all in a `QHBoxLayout()`.